### PR TITLE
Updated readmes according to restructured config

### PR DESCRIPTION
**Pull request by theory-in-progress**
### Description

- Updated the readmes of the specific templates according to the restructured configs

### CODE REVIEW

File: src/templates/template-text-classification/README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ torchrun \
 ```sh
 torchrun \
   --nproc_per_node #:::= it.nproc_per_node :::# \
-  main.py \
+  main.py config.yaml\
```
Comment (vfdev-5) [inline src/templates/template-text-classification/README.md:97]:
```suggestion
  main.py config.yaml \
```

**Review by vfdev-5 — COMMENTED** [review]
@theory-in-progress can we refactor a bit the common text from the README ?

File: src/templates/template-text-classification/README.md (hunk 1 above)
Comment (theory-in-progress) [reply src/templates/template-text-classification/README.md:97]: Yeah, we can actually!

**Review by vfdev-5 — APPROVED** [review]
Great! Thanks @theory-in-progress

**Pull request merged**